lib/app-router/handlers/enable-draft.ts

```diff
@@ -53,7 +57,9 @@ export async function enableDraftHandler(
     aud = vercelJwt.aud;
   }
 
-  if (bypassToken !== process.env.VERCEL_AUTOMATION_BYPASS_SECRET) {
+  // hobby Vercel accounts may not have a VERCEL_AUTOMATION_BYPASS_SECRET, so we fallback to checking the value against the CONTENTFUL_PREVIEW_SECRET
+  // env var, which is supported as a workaround for these accounts
```

Comment:
super nit: great note added here 👍 and think its fine to mention 'hobby vercel accounts', but as a broader note could say "certain vercel account tiers may not support a VERCEL_AUTOMATION_BYPASS_SECRET ..." or something to decouple from the word `hobby` in case that ever changes or they expand the tiers.

Reply:
i agree, good point! will tweak the language a bit